### ISSUE TITLE
Failed to read image format on less than 4k png files

### ISSUE THREAD

**Issue by Crackheadlegend** [colmap/colmap#1780]
**Describe the bug**
Running colmap through terminal trying to scan 444 png files all with a resolution of 3240 x 2160. Every image file fails to read.

**To Reproduce**
I'm installing and running colmap through the command line using the following commands
python scripts/colmap2nerf.py --colmap_matcher exhaustive --run_colmap --aabb_scale 16BMD

**Expected behavior**
I was expecting the files to analyze properly. Is there anywhere in documentation that clearly states what formats, data sizes, & resolutions are acceptable inputs? For some reason I can't seem to find accepted file formats and these would be very helpful details to have.

**Screenshots**
![image](https://user-images.githubusercontent.com/58893657/215307277-7f5b3887-8d75-4fbc-904e-8bf52d1d4348.png)

**Environment:**
 - OS: [e.g. Windows 10]
 - COLMAP Version [3.7]
 - Capture Device [BMD URSA 12k]

**Comment by ahojnnes** [comment]
If I interpret your screenshot correctly, then the images have 64 bit color depth per color channel? colmap only supports 8bits per channel. You could potentially convert the image to 8bit as a preprocessing step before feeding them to colmap.

**Comment by lumalav** [comment]
Hi, I have the same problem. But, I am not sure I understand the suggestion. How can I convert the images programmatically?

**Comment by simonnxren** [comment]
> If I interpret your screenshot correctly, then the images have 64 bit color depth per color channel? colmap only supports 8bits per channel. You could potentially convert the image to 8bit as a preprocessing step before feeding them to colmap.
yeah, it is the bit depth causing the issue. But I got another problem after converting to 8bit depth png, the reading camera# becomes less after i convert to 8bit png from 16bit.  so my question is what is the best way to obtain the images? or what is the best images source

**Comment by sam31046** [comment]
> @ahojnnes yeah, it is the bit depth causing the issue. But I got another problem after converting to 8bit depth png, the reading camera# becomes less after i convert to 8bit png from 16bit. so my question is what is the best way to obtain the images? or what is the best images source

In my case, I've checked png files and they're 8 bits, it failed. Then I converted then into jpg and it ran without error. For your question, I think the best image is having some features or textures in the background of your object, so that it can map those feature, I think.
